compiler/ir/backend.common/src/org/jetbrains/kotlin/backend/common/serialization/KotlinIrLinker.kt

```diff
@@ -294,6 +364,7 @@ abstract class KotlinIrLinker(
             ?: error("findDeserializedDeclaration: property descriptor $propertyDescriptor} is not present in propertyTable after deserialization}")
     }
 
+    // TODO: This is Native specific. Eliminate me.
```

Comment:
I agree that this method should be abstract

Reply:
I am going to eliminate the remaining native and js specific pieces from the common code in another fix.